apps/staking/src/app/api/v1/locked_accounts/route.ts

```diff
@@ -76,12 +77,12 @@ export async function GET(req: NextRequest) {
       const lock = await stakingClient.getUnlockSchedule(position, true);
       return {
         custodyAccount: custodyAccount.address.toBase58(),
-        actualAmount: Number(custodyAccount.amount),
+        actualAmount: Number(custodyAccount.amount) / FRACTION_PRECISION,
```

Comment:
You might want to use this utility instead, which will avoid precision issues from division by actually just formatting the string rather than dividing by the precision: https://github.com/pyth-network/pyth-crosschain/blob/main/apps/staking/src/tokens.ts#L3-L16

Reply:
nice one, used it.